applications/nucleo_io/targets/freertos.armv6m.st-stm32f091rc-nucleo-dev-board/ServoConsumer.hxx

```diff
@@ -0,0 +1,135 @@
+#ifndef _APPLICATIONS_IO_BOARD_TARGET_SERVOCONSUMER_HXX_
+#define _APPLICATIONS_IO_BOARD_TARGET_SERVOCONSUMER_HXX_
+
+#include "config.hxx"
+#include "os/MmapGpio.hxx"
+
+const unsigned servo_ticks_0 = configCPU_CLOCK_HZ * 1 / 1000;   // 48k
+const unsigned servo_ticks_180 = configCPU_CLOCK_HZ * 2 / 1000; // 96k
+
+uint8_t dummy_var[1] = {0x0};
+constexpr const MmapGpio8 DUMMY_GPIO(
```

Comment:
There is freertos_drivers/common/DummyGPIO.hxx
to see how to use https://github.com/bakerstu/openmrn/commit/ee860add363ec46af28bed56e894b9c97772f972